automation_tools/__init__.py

```diff
@@ -1832,68 +1859,68 @@ def sync_capsule_tools_repos_to_upgrade(admin_password=None):
     run(initials+'capsule content synchronize --id {0}'.format(capsule_id))
     # Create product capsule
     run(initials + 'product create --name capsule6_latest '
-        '--organization Default_Organization')
+        '--organization {0}'.format(org))
```

Comment:
The quoting because the space could be avoided if wrapping {0} into " like `"{0}"` :smile: